src/System.Threading/tests/AutoResetEventTests.cs

```diff
@@ -37,6 +37,14 @@ public void SetReset()
         }
 
         [Fact]
+        public void WaitHandleWait_Invalid()
+        {
+            Assert.Throws<ArgumentNullException>(() => WaitHandle.WaitAll(null));
```

Comment:
Just a heads up. I added a fix for the missing parameter name over in dotnet/coreclr#2889